### DISCUSSION TITLE
DateTime filter

### DISCUSSION THREAD

**Discussion by helander** [pocketbase/pocketbase#1923] [Q&A]
I have been trying to apply a filter on a List operation where the filter is set to be equal to a DateTime value, but without success. What is the proper format of the filter string for an operation using DateTime values?

**Comment by ganigeorgiev** [comment]
The full datetime value is in the format `Y-m-d H:i:s.uZ` (eg. `2022-02-26 17:37:00.123Z`).

If you don't want to specify an exact time or to target a day/period, you can use also a "range" style filter with only part of the above layout format, eg.:
```
myDateField >= '2022-02-01 10:00' && myDateField <= '2022-02-10 19:00'
```

**Please note that dates are always stored in UTC!**

You can also check https://github.com/pocketbase/pocketbase/discussions/1791#discussioncomment-4881369

If the above doesn't help, please share a code sample how are you specifying the date filter.

**Reply by helander** [reply]
Thank you very much, I got it to work now. It was the quotes that was missing.
